pkg/resource/uri.go

```diff
@@ -21,23 +21,35 @@ import (
 )
 
 var (
+	uriPrefixes = map[pb.ResourceType]string{
```

Comment:
What do you thinking about mergign `uriPrefixes` and `uriPatterns` into a single map with the value being a struct containing the prefix and regex pattern?